bitcoinspv/relayer.go

```diff
@@ -13,27 +13,18 @@ import (
 
 // Relayer manages the Bitcoin SPV relayer functionality
 type Relayer struct {
```

Comment:
Let's disable fieldalignment here as well. This is a singleton, so readability is more important

Reply:
done